doc/api/http.md

```diff
@@ -538,8 +538,11 @@ srv.listen(1337, '127.0.0.1', () => {
 ### request.abort()
 <!-- YAML
 added: v0.3.8
+deprecated: REPLACEME
 -->
 
+> Stability: 0 - Deprecated: Use [`request.destroy()`][] instead.
```

Comment:
1. Missing bottom reference.
2. Maybe `deprecations.md` also needs to be updated with a new section.

Reply:
@vsemozhetbyt sorry, I don't know what either of those two mean...

Reply:
@ronag 1. means that the link to ``[`request.destroy()`][]`` is not defined at the bottom of the file. For 2., I think that might become clear by looking at the contents of `deprecations.md`?

Reply:
Fixed 1. I don't know about 2 even after looking at the content.

Reply:
@ronag Something like https://github.com/nodejs/node/commit/ed24c19002379063ce100037bfff7ca2da5b6cc8#diff-2d795ff9a4ce61626915fd22f1684bb3, with `Type: Documentation-only` and `DEP0XXX` as a placeholder that is replaced when this is landed

Reply:
Thanks! Fixed.